packages/drawer/__tests__/drawer.spec.ts

```diff
@@ -0,0 +1,343 @@
+import { mount } from '@vue/test-utils'
+import Drawer from '../src/index.vue'
+import Button from '../../button/src/button.vue'
+
+const _mount = (template: string, data, otherObj?) =>
+  mount({
+    components: {
+      [Drawer.name]: Drawer,
+      [Button.name]: Button,
+    },
+    template,
+    data,
+    ...otherObj,
+  })
+
+const sleep = (time = 250) => new Promise(resolve => setTimeout(resolve, time))
+export const timeout = async (fn, time = 250) => {
+  await sleep(time)
+  fn()
+}
```

Comment:
I will recommend you to use `jest.useFakeTimer` instead of `sleep`  check this out: [jest.useFakeTimers](https://jestjs.io/docs/en/jest-object#jestusefaketimersimplementation-modern--legacy)

Reply:
ok!